etc/schema.json

```diff
@@ -202,79 +202,7 @@
-                    "properties": {
-                        "affected_product": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "attack_target": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "created_at": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "deployment": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "former_category": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "malware_family": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "policy": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "signature_severity": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "tag": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "updated_at": {
-                            "type": "array",
-                            "minItems": 1,
-                            "items": {
-                                "type": "string"
-                            }
-                        }
-                    },
-                    "additionalProperties": false
+                    "additionalProperties": true
```

Comment:
Makes sense as this is free-form data from the rule language.  Could be anything.

Reply:
For our SV it might make sense to be stricter, so we can spot issues?

Reply:
I do not think it is an issue to use something which is not in this list, as the ET rules do in https://github.com/OISF/suricata-verify/pull/1051

Reply:
> For our SV it might make sense to be stricter, so we can spot issues?

I think this would require a separate schema for S-V.  The idea of the schema is to produce something re-usable for documentation and code generation purposes, so it doesn't make sense to limit this to only the data that is seen in Suricata-Verify when it really is free-form data.

Reply:
I think we should still include the commonly used fields like we have currently, but do allow additional fields.